### ISSUE TITLE
caffe compile problem about the Version of OpenCv

### ISSUE THREAD

**Issue by chr10003566** [ShuangLI59/person_search#71]
CMake Warning at /home/tj/opencv-2.4.13.3/cmake/OpenCVConfig.cmake:165 (message):
  Found OpenCV Windows Pack but it has not binaries compatible with your
  configuration.

  You should manually point CMake variable OpenCV_DIR to your build of OpenCV
  library.
Call Stack (most recent call first):
  cmake/Dependencies.cmake:62 (find_package)
  CMakeLists.txt:30 (include)

CMake Error at cmake/Dependencies.cmake:62 (find_package):
  Found package configuration file:

    /home/tj/opencv-2.4.13.3/cmake/OpenCVConfig.cmake

  but it set OpenCV_FOUND to FALSE so package "OpenCV" is considered to be
  NOT FOUND.
Call Stack (most recent call first):
  CMakeLists.txt:30 (include)

I have done some solutions from the Internet，But it didn't work. Did the project have some constraints about the version of OpenCV? Would you like to tell me how to solve the problem? Thank you so much

**Comment by Cysu** [comment]
I'm not sure but could you please try to upgrade to OpenCV >= 3.0?

**Comment by chr10003566** [comment]
@Cysu Thank you for your reply. Yesterday, I have solved this problem. I found that your project had no constraint about the version of OpenCV. I guessed that the multiple version of OpenCV caused this problem. I uninstalled one of the OpenCV. The problem has been solved. I am trying to train my own datasets. I may have some other questions to ask for your help after. Thank you a lot for your contribution.
